history/cultures/orc_group.txt

```diff
@@ -30,11 +24,15 @@
 	discover_innovation = innovation_motte 
 	#
 	discover_innovation = innovation_city_planning
+	#
+	discover_innovation = innovation_wolfrider
 }
 508.1.1 = {
 	discover_innovation = innovation_catapult 
 	#
 	discover_innovation = innovation_currency_01
+	#
+	discover_innovation = innovation_wyvern
```

Comment:
There are 3 orc cultures that may have wyverns because they ventured with Thrall: Durotaric, Frostwolf, Warsong.